tests/test_analysis_tools.py

```diff
@@ -350,3 +360,170 @@ def test_process_rmsf_results(rmsd_functions, mock_plt_savefig, mock_savetxt, pl
 
     else:
         mock_plt_savefig.assert_not_called()
+
+
+################### SASA ######################
```

Comment:
Can you separate these into separate files instead of putting these separators?

Reply:
done - I organized new test files by "test_analysis_{SOME_FXNS}", so it'd be easy to find with so many test files